plothist/plotters.py

```diff
@@ -9,194 +9,46 @@
-        Label to display, default is variable name. Latex supported by surrounding the label with $label$.
-    log : bool
-        True if plot in logscale, default is False
-    legend_location : str
-        Default is best
-    legend_ncols : int
-        Default set to 1
-    docstring : str
-        Default is empty
 
-    Parameters
-    ----------
-    variables : list
-        A list of variable names to be registered.
-    path : str, optional
-        The path to the variable registry file (default is "./variable_registry.yaml").
-    reset : bool, optional
-        If True, the registry will be reset for all variables (default is False).
-
-
-    """
-
-    if not os.path.exists(path):
-        with open(path, "w") as f:
-            pass
-
-    with open(path, "r") as f:
-        variable_registry = yaml.safe_load(f)
-        if variable_registry is None:
-            variable_registry = {}
-
-        for variable in variables:
-            if variable not in variable_registry.keys() or reset:
-                variable_registry.update(
-                    {
-                        variable: {
-                            "name": variable,
-                            "bins": 50,
-                            "range": ["min", "max"],
-                            "label": variable,
-                            "log": False,
-                            "legend_location": "best",
-                            "legend_ncols": 1,
-                            "docstring": "",
-                        }
-                    }
-                )
-
-    with open(path, "w") as f:
-        for key, value in variable_registry.items():
-            yaml.safe_dump({key: value}, f, sort_keys=False)
-            f.write("\n" * 2)
-
-
-def get_variable_from_registry(variable, path="./variable_registry.yaml"):
-    """
-    This function retrieves the parameter information for a variable from the variable registry file specified by the 'path' parameter.
-    It loads the variable registry file and returns the dictionary entry corresponding to the specified variable name.
 
-    Parameters
-    ----------
-    variable : str
-        The name of the variable for which to retrieve parameter information.
-    path : str, optional
-        The path to the variable registry file (default is "./variable_registry.yaml").
-
-    Returns
-    -------
-    dict
-        A dictionary containing the parameter information for the specified variable.
-
-    See also
-    --------
-    create_variable_registry
-    """
-
-    if not os.path.exists(path):
-        if path == "./variable_registry.yaml":
-            raise RuntimeError("Did you forgot to run create_variable_registry()?")
-
-    with open(path, "r") as f:
-        variable_registry = yaml.safe_load(f)
-        return variable_registry[variable]
-
-
-def update_variable_registry(
-    variable_key, x_min, x_max, path="./variable_registry.yaml"
+def create_comparison_figure(
+    figsize=(6, 4),
```

Comment:
`figsize` should be set by the style, not hard coded. None should be the default value, writing in the documentation that it is defined in the style file by default.